src/Adapters/WebForms/RedirectHelper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+using DotVVM.Framework.Routing;
+
+#if NETFRAMEWORK
+using System.Web.Routing;
+#endif
+
+namespace DotVVM.Framework.Hosting
+{
+    public static class DotvvmRequestContextExtensions
+    {
+        public static void RedirectToRouteHybrid(this IDotvvmRequestContext context, string routeName, object routeValues = null, string urlSuffix = null, object query = null)
```

Comment:
I'd prefer to name it something like `RedirectToWebFormsRoute`, or at least name the class as WebForms...Extensions. The control is fine at will always have the `webforms:` prefix.

Reply:
Let's rename to `WebFormsExtensions` and make this method obsolete on .NET Core branch.